jobspec/test-fixtures/tg-scaling-policy.hcl

```diff
@@ -9,7 +9,7 @@ job "elastic" {
         foo = "bar"
         b   = true
         val = 5
-        f   = .1
```

Comment:
this looks strange, doesn't it?

Reply:
Yeah, I think this is just a ~~bug~~ feature in our `hclfmt` unfortunately. 

```bash
$ cat test.hcl 
foo {
  bar = .1
  baz = 2
}
$ hclfmt test.hcl > test.fmt.hcl
$ diff test.hcl test.fmt.hcl 
2c2
<   bar = .1
---
>   bar =.1
```

Reply:
nice feat ;)

Reply:
Fixed in: https://github.com/hashicorp/hcl/pull/382